### ISSUE TITLE
Accessibility: reflow

### ISSUE THREAD

**Issue by corrideat** [okTurtles/group-income#2746]
### Problem

The app fails badly at the [reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) requirement (zooming in at 400%), which is a WCAG AA requirement. To clarify, the image below shows the entirety of the content window.

![Reflow](https://github.com/user-attachments/assets/125f27bf-c6e2-44d4-b6d1-7a9de2106a37)

### Solution

Correct stylesheets to render well even at a 400% zoom level (and possibly beyond)

**Comment by corrideat** [comment]
These are some style rules being applied to the `:root`, `body` or `html` selectors that break accessibility:

```css
height: 100%;
max-height: 100%;
overflow: hidden;
min-width: 19rem;
```
